common/live/prd/infra/dev/sgrundell/terraform.tfvars

```diff
@@ -0,0 +1,25 @@
+# ↓ Module metadata
+
+terragrunt = {
+  terraform {
+    source = "/project/modules//gcp-project"
+  }
+  dependencies {
+    paths = ["../zone"]
+  }
+  include = {
+    path = "${find_in_parent_folders()}"
+  }
+}
+
+# ↓ Module configuration (empty means all default)
+
+project_name    = "dev-sgrundell"
```

Comment:
Is `sgrundell` Steve's username?

Reply:
Yes, his @rtf.org account.

Reply:
People's email addresses often don't match usernames they use: `sergey@` vs `natarajaya`, `javi@` vs `jhernandez`...

Reply:
Yeah, I double checked with Steve and he is fine with this.